c2corg_ui/static/js/utils.js

```diff
@@ -123,6 +123,26 @@ app.utils.formatDate = function(date) {
 
 
 /**
+ * check that the start date if before end date.
+ * Return true is start > end
+ * @export
+ * @param {Object} outing
+ * @return {(boolean | undefined)}
```

Comment:
We had a function where we actually wanted to return in one case a `string` and else nothing. So we wrote `string | undefined`, in this function actually i may want `true` or nothing, but `false` may be interpreted as such :P